components/studio/charts/apps/rstudio/chart/values.yaml

```diff
@@ -9,9 +9,26 @@ app:
     requests:
       memory: 1024Mi
       cpu: 1.0
+      ephemeral-storage: 10Mi
     limits:
       memory: 1024Mi
       cpu: 1.0
+      ephemeral-storage: 200Mi
+  
+  image: ghcr.io/scilifelabdatacentre/serve-rstudio:latest
+
+  port: 8787
+
+  securityContext:
+    allowPrivilegeEscalation: false
+    privileged: false
+    runAsNonRoot: true
+    runAsUser: 1000
+    readOnlyRootFilesystem: true
```

Comment:
This is a good thing to have, but we will likely need to either mount one or more emptydirs, or simply remove it, to have a usable (for the users) image.